pandas/io/formats/style.py

```diff
@@ -958,6 +958,8 @@ def background_gradient(
         axis=0,
         subset=None,
         text_color_threshold=0.408,
+        vmin=None,
```

Comment:
Can you annotate the new parameters?

Reply:
I got it! you mean the type-hints annotation right?

Reply:
Yes that is correct

Reply:
done! :smile: